src/types.ts

```diff
@@ -396,6 +403,47 @@ export enum Statistic {
     KILL_DEATH_RATION = 'kdratio',
 }
 
+export interface AppGrants {
+    banlist: BanListAppGrant[],
+    server: ServerAppGrant[],
+}
+
+export interface BaseResource {
+    id: string,
+    identifier: string,
+    object_id: string,
+}
+
+export interface ServerResource extends BaseResource {
+    gameserver_id: string,
+}
+
+export interface RawBanListAppGrant {
+    created_at: string,
+    resource: BaseResource,
+}
+
+export interface RawServerAppGrant {
+    created_at: string,
+    resource: ServerResource,
+}
+
+export interface RawAppGrants {
```

Comment:
TThis type, and all the other raw types, should live in the src/internal/got/client.ts file and not be exported. They are for internal use only :)